Code/GraphMol/ROMol.h

```diff
@@ -175,6 +235,42 @@ class ROMol : public RDProps {
   //@}
   //! \endcond
 
+  //! C++11 Range iterator
+  /*!
+    <b>Usage</b>
+    \code
+      for(auto atom : mol.atoms()) {
+         atom->getIdx();
+      };
+    \endcode
+   */
+  
+  CXXAtomIterator<MolGraph, Atom*> atoms() {
+        return { &d_graph };
+  }
+
+  CXXAtomIterator<const MolGraph, Atom*const> atoms() const {
```

Comment:
I admit that I've never gotten my brain around "const pointer to const object" notation, but I think `Atom *const` should be `const Atom *const`.

Reply:
I'll see if that compiles :)

Reply:
Yeah, "ask the compiler" is generally my solution too.